hw/dv/data/dsim/dsim.hjson

```diff
@@ -0,0 +1,122 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "{job_prefix} dsim"
+  build_ex:   "dsim"
+  run_cmd:    "{job_prefix} {build_ex}"
+
+  // Indicate the tool specific helper sources - these are copied over to the
+  // {tool_srcs_dir} before running the simulation.
+  tool_srcs:  ["{proj_root}/hw/dv/tools/dsim/*"]
+
+  build_opts: [
+               "-work {build_dir}/dsim_out",
+               "-genimage image",
+               "-sv",
+
+               // UVM
+               "+incdir+{UVM_HOME}/src",
+               "{UVM_HOME}/src/uvm_pkg.sv",
+               "+acc+rwb -sv_lib {DSIM_HOME}/lib/libuvm_dpi.so",
+
+               "-timescale 1ns/1ps",
+               "-f {sv_flist}",
+               "+incdir+{build_dir}",
+               // Suppress following DSim errors and warnings:
+               //   EnumMustBePositive - UVM 1.2 violates this (b/150231731)
+               "-suppress EnumMustBePositive",
+              ]
+
+  run_opts:   [
+               "-work {build_dir}/dsim_out",
+               "-image image",
+               // UVM DPI
+               "+acc+rwb -sv_lib {DSIM_HOME}/lib/libuvm_dpi.so",
+               "+ntb_random_seed={seed}",
+               "+UVM_TESTNAME={uvm_test}",
+               "+UVM_TEST_SEQ={uvm_test_seq}"]
+
+  // Coverage related.
+  cov_db_dir: "{scratch_path}/coverage/{build_mode}.vdb"
```

Comment:
All the coverage stuff appears to be specific to VCS. Perhaps make these empty vars and add TODO?

Reply:
Done